chainerrl/agents/categorical_double_dqn.py

```diff
@@ -24,10 +21,16 @@ def _compute_target_values(self, exp_batch):
         batch_rewards = exp_batch['reward']
         batch_terminal = exp_batch['is_state_terminal']
 
-        with chainer.using_config('train', False), state_kept(self.q_function):
```

Comment:
Is `chainer.using_config('train', False)` not needed here: https://github.com/chainer/chainerrl/pull/436/files#diff-84d12b0647363192349d452beaeef7d6R32?

Reply:
In double DQN, we don't get rid of it...